noetic/distribution.yaml

```diff
@@ -5424,6 +5424,23 @@ repositories:
       url: https://github.com/swri-robotics/mapviz.git
       version: master
     status: developed
+  marine_msgs:
+    doc:
+      type: git
+      url: https://github.com/apl-ocean-engineering/marine_msgs.git
+      version: main
+    release:
+      packages:
+      - marine_acoustic_msgs
+      - marine_sensor_msgs
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/CCOMJHC/marine_msgs-release.git
```

Comment:
I'm definitely not familiar with the release process ... is there a reason to use both the apl-ocean-engineering repo and the CCOMJHC version in this file?

Reply:
So the way that this works is that there has to be two repositories: one is the source repository, and where the work is generally done.  The second repository is what we call the `release` repository, and is where bloom pushes all kinds of metadata related to building packages.

Those two repositories can reside anywhere you can host a git repository, though we typically prefer github.com for some of the other automation that we can do there.  As far as the GitHub organization is concerned, the two repositories can be in separate organizations, as it is here.  For Noetic, at least, we don't have a preference, so this should be sorted out between @lauralindzey and @rolker .

Reply:
Thanks for the explanation! I have no objection to the release repository living under the CCOMJHC org.